packages/gatsby-plugin-now/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "gatsby-plugin-now",
+  "version": "1.2.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "./build.sh",
+    "test-integration-once": "jest"
+  },
+  "files": [
+    "index.js",
+    "gatsby-node.js"
+  ],
+  "peerDependencies": {
+    "gatsby": ">=2.0.0"
+  },
+  "devDependencies": {
+    "jest": "^24.9.0"
```

```suggestion
    "jest": "24.9.0"
```

Let's use exact versions for deps